programs/dual-farming/src/calculator/mod.rs

```diff
@@ -0,0 +1,40 @@
+use crate::*;
```

Comment:
Could you check whether there are new updates from step-finance reward pools?

Reply:
removed logging following step-finance reward-pool updates